content/docs/user-guide/setup-dropbox-remote.md

```diff
@@ -0,0 +1,132 @@
+# Setup a Dropbox DVC Remote
+
+In this guide we explain the existing ways to setup Dropbox
+[remote storage](/doc/command-reference/remote) for your <abbr>DVC
+projects</abbr>, along with the different benefits each one brings.
+
+## Quick start
+
+To start using a Dropbox remote, you only need to add it with a
+[valid URL format](#url-format). Then use any DVC command that needs to connect
+to it (e.g. `dvc pull` or `dvc push` once there's tracked data to synchronize).
+For example:
+
+```dvc
+$ dvc add data
+...
+$ dvc remote add --default myremote \
+                           dropbox://my-directory
+$ dvc push
+
+Go to the following link in your browser:
+
+    https://www.dropbox.com/oauth2/authorize?response_type=code&client_id=<client-id>&token_access_type=offline # ... copy this link
+
+Enter verification code: # <- enter resulting code
+```
+
+See [Authorization](#authorization) for more details.
+
+## URL format
+
+You can indicate any directory name that will be created under the
+`DROPBOX_HOME`, e.g. `DROPBOX_HOME/dvc/my-project`. Make sure this name is not
+already used for another project to keep things separated. The URL is formed as
+follows: `dropbox://dvc/my-project`. It does not need to be an existing
+directory.
```

Comment:
Just inverting the sentence order (and copy edits):

```suggestion
The URL is formed as follows: `dropbox://dvc/my-project`. You can indicate any
directory path (it will be created if it doesn't exist) under the
`DROPBOX_HOME`, e.g. `DROPBOX_HOME/dvc/my-project`. Make sure this directory is
not already used by another project, to keep things separate. 
```

Reply:
But what is `DROPBOX_HOME`?

Reply:
`DROPBOX_HOME` is just a path to the Dropbox folder on the user filesystem. It is not an env variable though, just a concept.

Reply:
I see. Maybe don't put it in back quotes because it looks like an env var. Idea:

```md
under the Dropbox home folder, e.g. `<Dropbox-home>/dvc/my-project`
```